checkout-example/Controllers/WebhookController.cs

```diff
@@ -52,7 +52,8 @@ public async Task<ActionResult<string>> Webhooks(NotificationRequest notificatio
                 // Process notification asynchronously.
                 await ProcessNotificationAsync(container.NotificationItem);
 
-                return Ok("[accepted]");
+                // Return a 202 status with an empty response body
+                return StatusCode(202);
```

Comment:
Return `Accepted()`

Reply:
Yes, a lot better. Thanks.

I have just corrected it. If you confirm that's ok I will update all other folders in this branch. Tnx